integration-box/csv_import_via_http/import.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+import pandas as pd
+from importlib import import_module
+
+os.system(f"{sys.executable} -m pip install --user pytd==1.0.0")
+import pytd
+
+def main():
+  url     = os.getenv('url')
+  df      = pd.read_csv(url)
+
+  column_setting  = import_module(os.getenv('column_setting'))
```

Comment:
This is too tricky to achieve this task. How about making the file as YAML or JSON?